ui/src/test/java/edu/wpi/grip/ui/codegeneration/tools/PythonPipelineInterfacer.java

```diff
@@ -0,0 +1,391 @@
+package edu.wpi.grip.ui.codegeneration.tools;
+
+import org.opencv.core.KeyPoint;
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfKeyPoint;
+import org.opencv.core.MatOfPoint;
+import org.opencv.core.Point;
+import org.opencv.core.Size;
+import org.opencv.imgcodecs.Imgcodecs;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class PythonPipelineInterfacer implements PipelineInterfacer {
+  StringBuilder str;
+  BufferedWriter out;
+  static File codeDir = null;
+
+  static {
+    try {
+      codeDir = new File(PipelineGenerator.class.getProtectionDomain()
+          .getCodeSource().getLocation().toURI());
+    } catch (URISyntaxException e) {
+      e.printStackTrace();
+      fail("Could not load code directory");
+    }
+  }
+
+  public PythonPipelineInterfacer(String className) {
+
+    try {
+      Runtime.getRuntime().exec("cd " + codeDir.getAbsolutePath().toString());
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+    str = new StringBuilder();
+    str.append("import sys\n");
+    str.append("import cv2\n");
+    str.append("sys.path.insert(0, \'");
+    //File temp = new File("__init__.py");
+    // System.out.println(codeDir.toPath().toString());
+    str.append(codeDir.getAbsolutePath().toString());
+    str.append("\')\n");
+    str.append("import ");
+    str.append(className);
+    str.append("\n");
+    str.append("pipe = ");
+    str.append(className);
+    str.append(".");
+    str.append(className);
+    str.append("()\n");
+    try {
+      out = new BufferedWriter(new FileWriter("testing.py"));
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+  }
+
+  /* (non-Javadoc)
+   * @see edu.wpi.grip.ui.codegeneration.tools.PipelineInterfacer#setSource(int, java.lang.Object)
+   */
+  @Override
+  public void setSource(int num, Object value) {
+    str.append("val = ");
+    Class name = value.getClass();
```

Comment:
Should be called `type`. `name` is confusing. Should also use wildcard `Class<?>` instead of raw `Class`